guides/common/modules/proc_creating-an-operating-system-for-debian.adoc

```diff
@@ -14,15 +13,23 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts > Operating Systems*.
-. Click *Create Operating System* to create an operating system entry for Debian.
-. Enter `Debian 11` as *Name* for the operating system.
+. Click *Create Operating System* to create an operating system entry for {os_name}.
+. Enter `{os_name} {os_major}` as *Name* for the operating system.
 Choose a name as reported by Ansible, Puppet, or Salt as fact.
-. Set the *Major Version* to `11` and omit the *Minor Version*.
-. Set the *Release Name* to `bullseye` for Debian 11 or `focal` for Ubuntu 20.04.
+. Set the *Major Version* to `{os_major}` and omit the *Minor Version*.
+ifeval::["{os_name}" == "Ubuntu"]
++
+[CAUTION]
+====
+If you set a minor version, you cannot provision hosts running Ubuntu 22.04 using Autoinstall.
+{Project} uses the major version to assemble the path for the ISO image, for example `/var/www/html/pub/installation_media/{os_name}/{os_major}-x86_64/`.
```

Comment:
`/var/www/html/pub` is Katello-only.

Reply:
I would even suggest removing this line. Otherwise, if you want to keep an argument for the statement above, use the following:

```suggestion
Depending on the major/minor version, {Project} searches at a different path for the boot files in the Installation Media. Especially for Ubuntu/Debian, the path is different according to the operating system version. 
```

Reply:
I would even suggest removing this line. Otherwise, if you want to keep an argument for the statement above, I'd suggest the following:

```suggestion
Depending on the major/minor version, {Project} searches at a different path for the boot files in the Installation Media. Especially for Ubuntu/Debian, the path is different according to the operating system version. 
```

Reply:
Reworded.